assets/js/components/settings/SettingsActiveModules.js

```diff
@@ -40,7 +40,7 @@ export default function SettingsActiveModules() {
 		.sort( ( a, b ) => a.order - b.order );
 
 	return (
-		<Layout>
+		<Layout className="googlesitekit-layout--rounded">
```

Comment:
It may be a bit cleaner to add this class via a prop, e.g. a `rounded` boolean prop.

Reply:
👍